src/edu/csus/ecs/pc2/ui/EditProblemFrame.java

```diff
@@ -30,7 +30,7 @@
 
     private IInternalController controller;
 
-    private EditProblemPane problemPane = null;
```

Comment:
Confirm that the only change in this file was a rename of problemPane, thanks.

Reply:
Confirmed.